tests/toplevel_unit.c

```diff
@@ -57,6 +57,46 @@ DEFINE_TEST(range_contains) {
     roaring_bitmap_free(bm);
 }
 
+DEFINE_TEST(contains_multi) {
+    // create rbm with RLE conatainer form 0 1000
+    roaring_bitmap_t *bm = roaring_bitmap_from_range(0, 1000, 1);
+    // add array container from 66000
+    for (size_t i = 66000; i < 68000; i+=2) {
+        roaring_bitmap_add(bm, i);
+    }
+    // add bitset container from 20000
+    for (size_t i = 132000; i < 140000; i+=2) {
+        roaring_bitmap_add(bm, i);
+    }
+
+    size_t length = 10;
+    bool results[length];
+    bool expected_results[length];
+    uint32_t values[length];
+    for (size_t i = 0; i < length; i+=2) {
+        expected_results[i] = 1;
+        expected_results[i+1] = 0;
+    }
+    values[0] =  1000;   // 1
```

Comment:
We don't expect 1000 to be in the bitmap: `roaring_bitmap_from_range` inserts the range from `[start, end)` (exclusive of the top value).

Reply:
that’s it, I will fix this later